README

```diff
@@ -34,6 +34,83 @@ xtensa-byt-elf-objcopy -I binary -O elf32-xtensa-le -B xtensa image.bin image.bi
 
 xtensa-byt-elf-objdump -D image.bin.elf > image.dis.txt
 
+
+logger
```

Comment:
Should be sof-logger now. Can we also have these instructions in the sof-docs wiki too.

Reply:
@lgirdwood name updated